backend/models/core/EntityType.js

```diff
@@ -120,14 +123,15 @@ class EntityTypes {
     return this.find({ slug });
   }
 
-  static async create({ name, slug }) {
+  static async create({ name, slug, is_single_type }) {
     const db = new DB();
 
-    const insertEntitiesSQL = "INSERT INTO entity_types (slug, name) VALUES (:slug, :name)";
+    const insertEntitiesSQL = "INSERT INTO entity_types (slug, name, is_single_type) VALUES (:slug, :name, :is_single_type)";
 
     await db.executeStatement(insertEntitiesSQL, [
       { name: "slug", value: { stringValue: slug } },
       { name: "name", value: { stringValue: name } },
+      { name: "is_single_type", value: { booleanValue: is_single_type } },
```

Comment:
Instead of making it a boolean flag, let's make it into a string, as there are a couple more to come. 

For now, the values can be "collection" (default), or "singleton"
```suggestion
      { name: "variant", value: { stringValue: variant } },
```